### DISCUSSION TITLE
Is there a way to only activate macsimize for one of my screens?

### DISCUSSION THREAD

**Discussion by bigmcnuggets** [Ubiquitine/MACsimize6#1] [General]
Is there a way to only activate macsimize for one of my screens?

**Comment by Ubiquitine** [comment]
Currently there is no such functionality. I'm not sure I would ever work on this. Adjusting script functionality based on the output/screen is possible, but it is know to be buggy. 
I'm really looking forward to see KDE devs implement "virtual desktop per monitor" where each monitor has its own set of virtual desktops. That would make so much sense and make scripts like this work as it should.
